### ISSUE TITLE
API thing for active users of an instance

### ISSUE THREAD

**Issue by Cassolotl** [tootsuite/mastodon#10984]
### Pitch

An API thing for active users (MAU).

### Motivation

There are API things for user_count, status_count and domain_count on /about - but not for active users (MAU). That seems incongruous! Active users seems like a number that would be at least as useful as the number of accounts ever created, especially for something as fluid as social network activity. It shows a picture of the Mastodon network *right now*.

I am trying to learn a little bit about how to combine APIs and spreadsheets, and I really want to make something that automatically keeps track of how active the Mastodon branch of the fediverse is over time, but the total number is not really very useful for what I'm interested in.

For example, there are 2470 accounts on queer.party but only 409 of those accounts have been active in the past month. I want to play with numbers to find out which instances are particularly good at retaining members, and to see how active Mastodon is overall, see which instances are more active, etc.

**Comment by nightpool** [comment]
/api/v1/instance/activity has weekly active users, but I agree that the
monthly activity number that was added in
https://github.com/tootsuite/mastodon/pull/10232 should also be exposed

**Comment by Cassolotl** [comment]
I would still like to take advantage of this imaginary API thing. Please don't close it, bot!

**Comment by trwnh** [comment]
nodeinfo has an `activeMonth` measure, if that's enough?

**Comment by Cassolotl** [comment]
I don't know if that's enough, but I will try to look into it. :) Thanks!

**Comment by nemobis** [comment]
Yes, this is a duplicate of https://github.com/mastodon/mastodon/issues/874

**Comment by Cassolotl** [comment]
#874 was closed because it was implemented in 2018, but then it got removed again. This issue was started in 2019 after it was removed so I don't think it counts as a duplicate.

**Comment by nemobis** [comment]
Removed? I see it everywhere: https://mastodon.social/nodeinfo/2.0

**Comment by Cassolotl** [comment]
Oh you're right, I got confused - sorry! I'll close this issue. :)
